addons/popochiu/migration/migrations/popochiu_migration_6.gd

```diff
@@ -0,0 +1,97 @@
+@tool
+class_name PopochiuMigration6
+extends PopochiuMigration
+
+const VERSION = 6
+const DESCRIPTION = "Add AnimationPlayer nodes to all inventory items"
+const STEPS = [
+	"Add AnimationPlayer child nodes to existing inventory items",
+]
+
+
+#region Virtual ####################################################################################
+## This is code specific for this migration. This should return [code]true[/code] if the migration
+## is successful. This is called from [method do_migration] which checks to make sure the migration
+## should be done before calling this.
+func _do_migration() -> bool:
+	return await PopochiuMigrationHelper.execute_migration_steps(
+		self,
+		[
+			_add_animation_players_to_inventory_items
+		]
+	)
+
+
+#endregion
+
+#region Private ####################################################################################
+## Add AnimationPlayer nodes to all inventory items that don't already have one.
+func _add_animation_players_to_inventory_items() -> Completion:
+	var inventory_path := PopochiuResources.INVENTORY_ITEMS_PATH
+	if not DirAccess.dir_exists_absolute(inventory_path):
+		return Completion.IGNORED
+	
+	var dir := DirAccess.open(inventory_path)
+	if dir == null:
+		return Completion.IGNORED
+	
+	var any_item_updated := false
+	
+	dir.list_dir_begin()
+	var file_name := dir.get_next()
+	
+	while file_name != "":
+		if dir.current_is_dir():
+			var item_scene_path := inventory_path.path_join(file_name).path_join("inventory_item_%s.tscn" % file_name)
+			if FileAccess.file_exists(item_scene_path):
```

Comment:
As I mentioned in a previous comment, this could also be done by accessing the `popochiu_data.cfg` file.

Reply:
Done